apps/etterlatte-vedtaksvurdering/src/main/kotlin/no/nav/etterlatte/vedtaksvurdering/klienter/SamKlient.kt

```diff
@@ -81,11 +80,10 @@ class SamKlientImpl(
     ): List<Samordningsvedtak> {
         try {
             val response =
-                httpClient.get {
-                    url("$resourceUrl/api/vedtak")
+                httpClient.get("$resourceUrl/api/vedtak") {
```

Comment:
litt mer konsekvent Gjenny-måte å konstruere på

Reply:
Forøvrig testet denne løsningen mot Q2 på sak 1002203 som har samordningsmeldinger.